### ISSUE TITLE
Remove Webpack from examples?

### ISSUE THREAD

**Issue by i-schuetz** [rustwasm/wasm-bindgen#2405]
Is there a way to remove this third party from the examples, to provide a simpler entry to newcomers? I imagine that people interested in WASM don't necessarily have a web development background, or have worked with Webpack.

The maintainer(s) are also not familiar with it (see [this issue](https://github.com/rustwasm/wasm-bindgen/issues/2375)), so if if something comes up, nobody can fix it. 

Since I don't have experience with bundlers I can't give advice, but it seems possible? See e.g. https://markus.oberlehner.net/blog/goodbye-webpack-building-vue-applications-without-webpack/

**Comment by xtuc** [comment]
I would prefer to keep the webpack example around, but if it's simpler to hide it a bit more that would ok. I'm the one that introduced the it and didn't look after it for a while.

**Comment by i-schuetz** [comment]
You would like to keep it around because...? And are you able to help with the linked issue?

**Comment by alexcrichton** [comment]
I don't think it makes sense to remove webpack entirely, that's a feature of wasm-bindgen that it's compatible with webpack. I do think, though, that it may make more sense to transition examples to `--target web` by default and have a specific example for just webpack.

**Comment by i-schuetz** [comment]
>  that's a feature of wasm-bindgen that it's compatible with webpack

What does compatibility mean here? Is there any client side Javascript project that can't be used with webpack?

> have a specific example for just webpack

Agree! dedicated webpack examples would be better. The main examples should contain only the minimum to get it working. In the [linked issue](https://github.com/rustwasm/wasm-bindgen/issues/2375) we don't even know whether the problem is caused by webpack or wasm-bindgen. If it's possible to separate the examples, this kind of situation can't happen.

**Comment by alexcrichton** [comment]
By compatibility I mean there's an example of showing how to use wasm-bindgen and webpack together to stitch everything together.

**Comment by 0xorial** [comment]
Is wasm-bindgen intended to be used independently from wasm-pack/WasmPackPlugin ? wasm-pack has some annoying limitations so I am about to resort to use wasm-bindgen directly, but the examples in the docs give an impression that it is not how it is supposed to be used...
